config/defaults-config.json

```diff
@@ -108,5 +108,7 @@
   "CLEANUP_TELEGRAM": true,
   "CLEANUP_TIME_TG":"10",
   "CLEANUP_DATABASE": true,
-  "CLEANUP_TIME_DB":"10080"
+  "CLEANUP_TIME_DB":"10080",
+  "INLINESEARCH_LATITUDE":48.8582012,
+  "INLINESEARCH_LONGITIDE":2.2941477
```

Comment:
`s/LONGITIDE/LONGITUDE/` but this may be moot since coordinates should not be used for sorting.